api/src/public/login/style.css

```diff
@@ -40,10 +40,12 @@ input {
   line-height: 1.42857;
   border-radius: 4px;
   display: block;
-  margin-top: 2em;
+  margin: 2em auto;
   font-size: 1em;
   width: 302px;
+  max-width: calc(100% - 20px);
```

Comment:
Instead of using `calc` can we just add 20px to the horizontal padding?

Reply:
Actually it was the horizontal padding that was the problem :) But I switched `box-sizing` so it would calculate width including padding.